Makefile

```diff
@@ -25,13 +25,33 @@ build:
 	git submodule update --init --recursive
 	# Update RapidWright jars
 	# TODO: remove patch once https://github.com/Xilinx/RapidWright/issues/183 is solved
+
+	curl -O https://capnproto.org/capnproto-c++-0.8.0.tar.gz
+	tar zxf capnproto-c++-0.8.0.tar.gz
+	pushd capnproto-c++-0.8.0 && \
+		./configure && \
+		$(MAKE) check && \
+		sudo make install && \
+		popd
+
+	git clone https://github.com/capnproto/capnproto-java.git
+	pushd capnproto-java && \
+		$(MAKE) && \
+		sudo make install && \
+		popd
+
 	pushd ${RAPIDWRIGHT_PATH} && \
 		make update_jars && \
 		wget https://github.com/Xilinx/RapidWright/releases/download/v2020.2.5-beta/rapidwright_api_lib-2020.2.5-patch1.zip && \
 		unzip -o rapidwright_api_lib-2020.2.5-patch1.zip && \
 		popd
+	pushd third_party/prjoxide/libprjoxide && \
+		( curl --proto '=https' -sSf https://sh.rustup.rs | sh -s -- -y ) && \
+		source ${HOME}/.cargo/env && \
+		cargo install --path prjoxide --all-features && \
+		popd
 	# Build test suite
-	@$(IN_CONDA_ENV) mkdir -p build && cd build && cmake $(CMAKE_FLAGS) ..
+	@$(IN_CONDA_ENV) mkdir -p build && cd build && cmake $(CMAKE_FLAGS) -DPRJOXIDE=${HOME}/.cargo/bin/prjoxide ..
```

Comment:
The prjoxide var should not be required anymore

Reply:
I don't think `.cargo/bin` is in `PATH` at this point

Reply:
Unless adding `-DPRJOXIDE...` adds the var in the environment, the [`find_program`](https://github.com/SymbiFlow/fpga-interchange-tests/blob/d244771a4a618c9aa686f8e0f44698f1bcf4da8a/utils/utils.cmake#L12) should have failed if prjoxide couldn't be found by CMake

Reply:
yeah that code bypasses `find_program` when the var is passed, the other option to adding the var would be to override `PATH`. anyway it can definitely be removed once we use the Conda package that is properly on `PATH`.

Reply:
All right, I think we can merge this one and, as soon as the conda package is ready we can change this back.